js/getZip.js

```diff
@@ -0,0 +1,31 @@
+var zipCode;
+var zipCodeFlag = false;
```

Comment:
This seems a bit redundant...you can just check whether `zipCode` is set or not.